coalib/parsing/filters/UseStdinFilter.py

```diff
@@ -0,0 +1,31 @@
+def use_stdin_filter(bears, use_stdin='True'):
+    bears = filter_bears_by_stdin(bears)
+    return bears
+
+
+def filter_bears_by_stdin(bears, use_stdin='True'):
+    local_bears, global_bears = bears
+    local_bears = filter_section_bears_by_stdin(local_bears, use_stdin)
+    global_bears = filter_section_bears_by_stdin(
+        global_bears, use_stdin)
+    return local_bears, global_bears
+
+
+def filter_section_bears_by_stdin(bears, use_stdin='True'):
+    """
+    Filters the bears by use_stdin.
+
+    :param bears:       The dictionary of the sections as keys and list of
+                        bears as values.
+    :param use_stdin:   Value indicating whether filtered bears
+                        should use stdin
+    :return:            New dictionary with filtered out bears with
+                        use_stdin value specified in param.
+    """
+    new_bears = {}
+    stdin_values = set(stdin.lower() for stdin in use_stdin)
+    for section in bears.keys():
+        new_bears[section] = tuple(
+            bear for bear in bears[section]
+            if {stdin.lower() for stdin in bear.use_stdin} & stdin_values)
```

Comment:
@Makman2 
With `coala -B --filter-by use_stdin True`,
I'm getting the following error:

> type object 'VerilogLintBear' has no attribute 'use_stdin'

Bear object doesn't have use_stdin attribute. I'm not sure how to access a value from decorator's object.
Do you have any clues ?